src/Test/Perf/install.csx

```diff
@@ -0,0 +1,103 @@
+#load "util\assemblies.csx"
+#load "util\ngen.csx"
+
+using System;
+using System.Diagnostics;
+using System.IO;
+
+// If we're being #load'ed by uninstall.csx, set the "uninstall" flag.
+var uninstall = Environment.GetCommandLineArgs(1) == "uninstall.csx";
+var message = uninstall ? "Restoring previous copy of" : "Installing";
+
+// TODO: Use actual command line argument parser so we can have help text, etc...
+var sourceFolder = Args.Length == 1 ? Args[0] : @"C:\Roslyn\Binaries\Release";
+
+foreach (var processName in new[] { "devenv", "msbuild", "VBCSCompiler"})
+{
+    var processes = Process.GetProcessesByName(processName);
+    foreach (var p in processes)
+    {
+        Log($"Killing process \"{p.ProcessName}\", PID: {p.Id}");
+        p.Kill();
+    }
+}
+
+Log($"{message} Roslyn binaries to VS folder.");
+var devenvFolder = Path.GetFullPath(Environment.GetFolderPath(Environment.SpecialFolder.ProgramFilesX86), @"Microsoft Visual Studio 14.0\Common7\IDE\PrivateAssemblies");
```

Comment:
This is another place where a file having our constants could be used as `14.0` will change release to release.